pubspec.yaml

```diff
@@ -3,20 +3,22 @@ description: A personal finance managing app
 
 publish_to: "none" # Remove this line if you wish to publish to pub.dev
 
-version: "0.3.3+35"
+version: "0.3.4+0"
 
 environment:
   sdk: ">=3.1.3 <4.0.0"
 
 dependencies:
   auto_size_text: ^3.0.0
+  colornames: ^0.1.0
```

Comment:
It seems like `colornames` only supports English...

Reply:
Hello @sadespresso, sorry for the delay. I've been busy with life lately. Anyway, I've removed the `colornames` package as a dependency with my latest changes